tests/4.5/taskloop/test_taskloop_num_tasks.c

```diff
@@ -0,0 +1,61 @@
+//------------------------------------------- test_taskloop_num_tasks.c --------------------------------------//
+//
+// OpenMP API Version 4.5 September 2015
+//
+// This test checks the 'taskloop' directive with the 'num_tasks' clause specified.
+// The 'taskloop' construct parallelize loops with independent iterations by creating tasks. 
+// It allows for efficient parallel execution of loop iterations by distributing them among multiple threads. 
+// The 'num_tasks' clause variable ensures that the loop iterations are shared amoung created 'num_tasks'. 
+// i.e the loop iterations are divided amoung number of tasks (each num_task group is run by same the thread).   
+//----------------------------------------------------------------------------------------------------------//
+
+#include <stdio.h>
+#include <omp.h>
+#include "ompvv.h"
+
+#define NUM_THREADS 100
+#define NUM_TASKS 3
+
+int test_taskloop_num_tasks() {
+
+  int errors = 0;
+
+  long int all_thread_sum, real_sum = 0; 
+  long int var = 0; //This variable is shared with all the tasks.  
+  
+  int NUM = 100;
+
+   //get valid sum without openmp
+   for(int i = 0; i < NUM; i++) {
+     real_sum = real_sum + i;
+   }
+   
+   #pragma omp parallel num_threads(NUM_THREADS)
+   {
+      #pragma omp single
+      {
+	#pragma omp taskloop num_tasks(NUM_TASKS) 
+        for(int i = 0; i < NUM; i++) 
+	{
+	    #pragma omp atomic 
+	    var = var + i;
```

Comment:
Atomic operations on var do not check the behavior of num_task-- a better way would be to see if the num/num_tasks tasks were each executed by the same thread.

Reply:
https://github.com/SOLLVE/sollve_vv/pull/782/commits/24882279d1b6bd4bd0a2f3639ee75976d06dbf93

Reply:
I don't see a change to the check.